src/super_gradients/module_interfaces/exportable_detector.py

```diff
@@ -0,0 +1,399 @@
+import abc
+import copy
+import dataclasses
+from typing import Any
+from typing import Union, Optional, List, Tuple
+
+import onnxsim
+import torch
+from torch import nn, Tensor
+from torch.utils.data import DataLoader
+
+from super_gradients.common.abstractions.abstract_logger import get_logger
+from super_gradients.conversion.conversion_enums import ExportTargetBackend, ExportQuantizationMode, DetectionOutputFormatMode
+from super_gradients.conversion.onnx.nms import attach_onnx_nms
+from super_gradients.conversion.preprocessing_modules import CastTensorTo
+from super_gradients.conversion.tensorrt.nms import attach_tensorrt_nms
+from super_gradients.training.utils.export_utils import infer_format_from_file_name, infer_image_shape_from_model, infer_image_input_channels
+from super_gradients.training.utils.quantization.fix_pytorch_quantization_modules import patch_pytorch_quantization_modules_if_needed
+from super_gradients.training.utils.utils import infer_model_device
+from super_gradients.training.utils.quantization.selective_quantization_utils import SelectiveQuantizer
+
+logger = get_logger(__name__)
+
+__all__ = ["ExportableObjectDetectionModel", "AbstractObjectDetectionDecodingModule", "ModelExportResult"]
+
+
+class AbstractObjectDetectionDecodingModule(nn.Module):
+    @abc.abstractmethod
```

Comment:
Please add some docs.

Reply:
Fixed in e2d4fece90638c286336fcacf162bb86805da476